packages/teams-js/src/public/pages.ts

```diff
@@ -537,13 +580,21 @@ export namespace pages {
     }
 
     /**
-     * Checks if pages.backStack capability is supported currently
+     * Checks if the current application host supports the pages.backStack capability
+     * @returns true if the pages.backStack capability is enabled in runtime.supports.pages.backStack and
+     * false if it is disabled
      */
     export function isSupported(): boolean {
       return runtime.supports.pages ? (runtime.supports.pages.backStack ? true : false) : false;
     }
   }
 
+  /**
+   * @hidden
+   * Hide from docs
+   * ------
+   * Namespace to interact with the full-trust part of the SDK. Limited to 1P apps
```

Comment:
@erikadoyle is this a sufficient summary of full trust? I'm not sure if we have something more informative to use honestly.

Reply:
Good enough for now I think, esp. since its internal.